packages/manager/apps/web/client/app/hosting/website-coach/website-coach.html

```diff
@@ -0,0 +1,63 @@
+<div class="container-fluid px-5 mt-5">
+    <div class="row">
+        <div class="col-md-4">
+
+            <!-- Screenshot visualizer -->
+            <div class="website-coach-visualizer">
+                <img class="website-coach-visualizer__computer"
+                    src="/assets/images/website-coach-computer.png"
+                    alt="{{$ctrl.attachedDomain}}">
+                <div class="website-coach-visualizer__screenshot"
+                    data-ng-style="::{
+                        'background-image': 'url(\'data:image/png;base64,' + $ctrl.screenshot + '\')'
+                    }">
+                </div>
+            </div>
+        </div>
+        <div class="col-md-8">
+            <oui-back-button
+                data-on-click="$ctrl.goBack()"
+                data-heading="{{:: 'hosting_website_coach_title' | translate }}">
+            </oui-back-button>
+
+            <!-- One domain name -->
+            <div data-ng-if="!$ctrl.hasMultipleAttachedDomain">
+                <p>
+                    <span data-translate="hosting_website_coach_1dn_text1"></span>
+                    <strong data-ng-bind="$ctrl.attachedDomain"></strong>
+                    <span data-translate="hosting_website_coach_1dn_text2"></span>
+                    <strong data-ng-bind="$ctrl.user.email"></strong><sup>*</sup>
+                    <span data-translate="hosting_website_coach_1dn_text3"></span>
+                </p>
+            </div>
+
+            <!-- Multiple domain name -->
+            <div data-ng-if="$ctrl.hasMultipleAttachedDomain">
+                <p>
+                    <span data-translate="hosting_website_coach_mdn_text1"></span>
+                    <strong data-ng-bind="$ctrl.user.email"></strong><sup>*</sup>
+                    <span data-translate="hosting_website_coach_mdn_text2"></span>
+                </p>
+
+                <oui-field label="{{:: 'hosting_website_coach_mdn_label_domain_name' | translate }}" size="l">
+                    <oui-select name="attachedDomain"
+                        data-model="$ctrl.attachedDomain"
+                        data-items="$ctrl.attachedDomains">
+                    </oui-select>
+                </oui-field>
+            </div>
+
+            <oui-button class="mb-3"
+                data-variant="primary"
+                data-on-click="$ctrl.activateWebsiteCoach($ctrl.attachedDomain)">
+                <span data-translate="hosting_website_coach_button"></span>
+            </oui-button>
+
+            <p>
+                <sup>*</sup>
+                <small data-translate="hosting_website_coach_help_wrong_email"></small>
```

Comment:
`small` has a semantic meaning (for small prints), use CSS instead to write text smaller

Reply:
Updated with `.small` class